From d07659473bcf1e230cf0ad0b1ebb5cec2539c8c0 Mon Sep 17 00:00:00 2001
From: Alpha <charlieprashanth@gmail.com>
Date: Tue, 10 May 2022 22:37:05 +0530
Subject: [PATCH] commit1

---
 README.md | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/README.md b/README.md
index eb9d1f6..8dc934a 100644
--- a/README.md
+++ b/README.md
@@ -1 +1,4 @@
-# dragonball
\ No newline at end of file
+goku
+vegeta
+krilin
+
-- 
2.25.1

